CodyJones4235/PoopMap
# login is in place

#### Changes Made
1. Added file `login ` to `auth` directory.
1. Modified file `login ` to include `login ` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-creating-posts
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does login , then login should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
